test/e2e/egress_traffic_test.go

```diff
@@ -0,0 +1,43 @@
+package e2e
```

Comment:
you want to add the tag
// +build e2e

Reply:
done, thanks
we still need to update istio configs to make this test pass successfully: 
https://github.com/knative/serving/pull/3665#issuecomment-482295163